### PR TITLE
AKS Migration changes

### PR DESCRIPTION
**Pull request by amitchidrewar1301**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/bar-api/values.yaml [hunk 1]
```diff
@@ -8,24 +8,32 @@ postgresql:
     enabled: false
 
 java:
-  # Don't modify below here
   image: 'hmctspublic.azurecr.io/bar/api:latest'
   ingressHost: ${SERVICE_FQDN}
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: Make it something like this :
https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L3

File: charts/bar-api/values.aat.template.yaml [hunk 2]
```diff
@@ -5,3 +5,10 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  keyVaults:
```
Comment (adusumillipraveen) [inline charts/bar-api/values.aat.template.yaml]: You can move them back to values.yaml ?Not sure why u want to do aat .. which means preview wont work.

File: infrastructure/main.tf [hunk 3]
```diff
@@ -18,7 +18,7 @@ data "azurerm_key_vault" "bar_key_vault" {
   resource_group_name = "${local.vault_rg_name}"
 }
 
-data "azurerm_key_vault_secret" "s2s_secret" {
+data "azurerm_key_vault_secret" "S2S-SECRET" {
```
Comment (adusumillipraveen) [inline infrastructure/main.tf]: There is no reason for this rename , if you want to, change this as well https://github.com/hmcts/bar-apps/pull/280/files#diff-1031eeab417cd8afe77e7e005717d38dR45

File: charts/bar-api/values.preview.template.yaml [hunk 4]
```diff
@@ -5,3 +5,8 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  keyVaults:
+   "bar":
```
Comment (adusumillipraveen) [inline charts/bar-api/values.preview.template.yaml]: Indentation ?

File: charts/bar-api/values.yaml (hunk 1 above)
Comment (AnilDhawaleGitHub) [reply charts/bar-api/values.yaml]: its done

File: charts/bar-api/values.aat.template.yaml (hunk 2 above)
Comment (AnilDhawaleGitHub) [reply charts/bar-api/values.aat.template.yaml]: resolved

Comment (AnilDhawaleGitHub) [reply charts/bar-api/values.aat.template.yaml]: resolved

File: infrastructure/main.tf (hunk 3 above)
Comment (AnilDhawaleGitHub) [reply infrastructure/main.tf]: resolved

File: charts/bar-api/values.preview.template.yaml (hunk 4 above)
Comment (AnilDhawaleGitHub) [reply charts/bar-api/values.preview.template.yaml]: resolved

File: charts/bar-api/values.preview.template.yaml [hunk 5]
```diff
@@ -5,3 +5,8 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  keyVaults:
+        "bar":
```
Comment (adusumillipraveen) [inline charts/bar-api/values.preview.template.yaml]: Indentation seems wrong ?

File: charts/bar-api/values.yaml [hunk 6]
```diff
@@ -8,24 +8,31 @@ postgresql:
...
+    POSTGRES_PORT: 5432
     SPRING_LIQUIBASE_ENABLED: true
-
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://preprod-idamapi.reform.hmcts.net:3511
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-
     APPINSIGHTS_INSTRUMENTATIONKEY: fake-key
+    keyVaults:
+          "bar":
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: Is this indentation right?

File: charts/bar-api/values.preview.template.yaml [hunk 7]
```diff
@@ -5,3 +5,5 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  postgresql:
```
Comment (adusumillipraveen) [inline charts/bar-api/values.preview.template.yaml]: Indentation should be like here : https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.preview.template.yaml#L13

File: charts/bar-api/values.yaml [hunk 8]
```diff
@@ -8,24 +8,30 @@ postgresql:
...
-      resourceGroup: rpe-service-auth-provider
-      secrets:
-        - microservicekey-bar-api
   environment:
     SPRING_DATASOURCE_URL: "jdbc:postgresql://{{ .Release.Name }}-bar-db:5432/bar"
-    SPRING_DATASOURCE_USERNAME: bar
-    SPRING_DATASOURCE_PASSWORD: bar
-
+    POSTGRES_HOST: "{{ .Release.Name }}-postgresql"
+    POSTGRES_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: these 4 should move to preview template. please refer draft-store

**Pull request closed without merging**